app/views/rentals/_form.html.erb

```diff
@@ -1,64 +1,149 @@
+        </div>
+      </div>
+
+      <div class="form-group col-xs-12">
+        <div class="col-xs-offset-2 col-xs-10">
+          <%= f.submit class: "btn btn-primary", id: "TOC", disabled: "disabled" %>
+          <%= link_to 'Back', new_rental_path, class: "btn btn-default" %>
+        </div>
+      </div>
+    <% end %>
   </div>
-  <div class="form-group col-xs-12">
-    <div class="col-xs-offset-2 col-xs-10">
-      <%= f.submit class: "btn btn-primary", id: "TOC", disabled: "disabled" %>
-      <%= link_to 'Back', rentals_schedule_path, class: "btn btn-default" %>
-    </div>
+
+  <div class="tab-pane" id="multiple_day" role="tabpanel">
+    <%= form_for @rental, method: :post do |f| %>
+
+      <div class="form-group col-xs-12">
+        <%= f.label :type, 'Cart Type', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.select :item_type_id, options_from_collection_for_select(@item_types, "id", "name"), {}, class: "selectpicker form-control" %>
+        </div>
+      </div>
+
+      <div class="form-group col-xs-12">
+        <%= f.label :name, 'Cart Name', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.select :item_id, options_from_collection_for_select(@items.where(item_type_id: params["item_type_id"], deleted_at: nil).order(name: :asc), "id", "name"), {}, class: "selectpicker form-control" %>
+        </div>
+      </div>
+      <div class="form-group col-xs-12">
+        <%= f.label :user, 'User', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.select :user_id, options_from_collection_for_select(@users.order(last_name: :asc), "id", "full_name"), {}, class: "selectpicker form-control", onchange: "update_disclaimer(this)" %>
+        </div>
+      </div>
+
+      <div class="form-group col-xs-12">
+        <%= f.label :start_time, 'Start Date', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.date_field :start_time, value: @start_date, required: true, class: "form-control" %>
+        </div>
+      </div>
+
+      <div class="form-group col-xs-12">
+        <%= f.label :end_time, 'End Date', class: 'control-label col-xs-2' %>
+        <div class="col-xs-6">
+          <%= f.date_field :end_time, value: @start_date.next_day, required: true, class: "form-control" %>
+        </div>
+      </div>
+
+      <div class="form-group col-xs-12">
+        <div class="col-xs-offset-2 col-xs-6">
+          <div class="panel panel-primary">
+            <div class="panel-heading">
+              Terms and Conditions
+            </div>
+            <% @item_types.each do |type| %>
+              <div class="panel-body no-gutter" style="max-height: 10;overflow-y: scroll;">
+                <% hidden = '' %>
+                <% hidden = 'display:none; ' unless type == @item_types.first %>
```

Comment:
So, maybe we were talking about two different ways of accomplishing the same thing. Granted, you are reusing someone else's code, but reusing code presents a great opportunity to refactor. Might also be why my issues take forever to close...

![b1pdu](https://cloud.githubusercontent.com/assets/129601/17179576/b90a5f32-53e7-11e6-81e0-0a0153edad3b.gif)


Anyway, back to the task at hand! Aside from the remaining inline style (which could be factored out), I was suggesting adding a class to hide the elements.

```SASS
.hidden-element
  display: none
```

```RUBY
<%= content_tag :span,
          type.disclaimer,
          class: "disclaimer hidden-element",
          id: "disclaimer_#{type.id}" %>
```

The advantage to moving the style from inline up to a rental or global SASS/CSS file is that we can change the style once, and change the whole site! Further, the JS is just as easy, as it turns into an issue of adding and removing classes on the element.

Side note, this might be a bug, but changing the item type does not hide/unhide the disclaimer (in the previous implementation). Am I missing is, or was this something that was never implemented?